src/modules/AdvancedPaste/AdvancedPaste.FuzzTests/AdvancedPaste.FuzzTests.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<TargetFramework>net8.0-windows10.0.19041.0</TargetFramework>
```

Comment:
Do yo mind to add a readme on this project?
e.g. why need .net 8, what does it used for, link to overview of this onefuzz external doc (that's fine even we put internal link in case)

Reply:
Thanks, I have added the README with relevant links and a detailed description of the project in the latest commit.